proto/validator/accounts/v2/web_api.proto

```diff
@@ -18,6 +18,11 @@ service Wallet {
             body: "*"
         };
     }
+    rpc DefaultWalletPath(google.protobuf.Empty) returns (DefaultWalletResponse) {
```

Comment:
```suggestion
    rpc DefaultWallet(google.protobuf.Empty) returns (WalletResponse) {
```
I think the "Path" part doesn't align with the path. Also, can you use `WalletResponse`? 
That seems to make more sense to me.